platform/ios/src/MGLMapView.h

```diff
@@ -526,6 +526,13 @@ MGL_EXPORT IB_DESIGNABLE
  */
 @property(nonatomic, getter=isHapticFeedbackEnabled) BOOL hapticFeedbackEnabled;
 
+/**
+ A Boolean value that determines whether the updating pitch will also affect the altitude.
+ 
+ When this property is set to `YES`, pitch will work independently from altitude.
+ */
+@property(nonatomic, getter=isAltitudeUnaffectedByPitch) BOOL altitudeUnaffectedByPitch;
```

Comment:
I think the opposite `altitudeAffectedByPitch` might be less confusing? Also, we should specify the default value.

Reply:
I chose this naming for the purpose of defaulting to false. I'm definitely open to better naming. When you get altitude from the internal GL core zoom level, pitch is taken into account in the calculation. Same thing when you set zoom level from altitude. I want just the pure altitude and the pure pitch. Any suggestions?